src/utils/server-requests.js

```diff
@@ -271,42 +305,46 @@ export const updateProductQuantity = async (
   setIsUpdating,
   state,
   token,
-  type
+  type,
+  setToastMessage
 ) => {
   setIsUpdating(true);
-  if (product.qty >= product.avalQty && type == "increment") {
-    alert(`Can not than ${product.avalQty}`);
-    setIsUpdating(false);
-  } else {
-    try {
-      const res = await axios({
-        method: "post",
-        url: `/api/user/cart/${product._id}`,
-        data: {
-          action: {
-            type,
-          },
-        },
-        headers: {
-          authorization: token,
+
+  try {
+    const res = await axios({
+      method: "post",
+      url: `/api/user/cart/${product._id}`,
+      data: {
+        action: {
+          type,
         },
+      },
+      headers: {
```

Comment:
suggestion: to add axios interceptors for common headers to apis

Reply:
I will have to read about this too. 

Thank you for the suggestions.